Documentation/ColumnsConfig/CommonProperties/ItemsProcFunc.rst

```diff
@@ -9,62 +9,86 @@ itemsProcFunc
 
    :type: string (class->method reference)
    :Scope: Display / Proc.
-   :Types: :ref:`check <columns-check>`, :ref:`flex <columns-flex>`,
-      :ref:`radio <columns-radio>`
+   :Types: :ref:`check <columns-check>`, :ref:`select <columns-select>`, :ref:`radio <columns-radio>`
 
-   PHP function which is called to fill / manipulate the array with elements.
+   PHP method which is called to fill or manipulate the items array.
 
-   [classname]->[methodname]
+   VENDOR\\Extension\\UserFunction\\YourClass->yourMethod
```

Comment:
In general it is recommended to use the actual FQCN with `::class` instead and then concatenate the method:

```php
\VENDOR\Extension\UserFunction\YourClass::class . '->yourMethod'
```

Big advantage: with an IDE or similar places like these are also updated on refactoring.